src/commands/createFunction/FunctionListStep.ts

```diff
@@ -62,6 +66,10 @@ export class FunctionListStep extends AzureWizardPromptStep<IFunctionWizardConte
     }
 
     public async getSubWizard(context: IFunctionWizardContext): Promise<IWizardOptions<IFunctionWizardContext> | undefined> {
+        if (context.functionTemplate?.templateSchemaVersion === TemplateSchemaVersion.v2) {
```

Comment:
It actually makes more sense for this do be pushed in here. The `JobsListStep` actually depends on there being a function template to iterate its `jobs` array.